mock/py/mock.py

```diff
@@ -602,6 +602,11 @@ def running_in_docker():
             items = line.split(':')
             if 'docker' in items[2]:
                 return True
+    # For containers with cgroupv2
+    with open('/proc/self/mountinfo', encoding='utf8') as f:
+        for line in f:
+            if '/docker/containers/' in line:
```

Comment:
Would it make sense to check for e.g. `/etc/hosts` presence on the same line?  Just in case the /docker/containers mount point existed even in a non-container case?